travel_platform/travel_app/views.py

```diff
@@ -9,22 +9,35 @@
 
 from .models import Query
 
+import requests
+
 
 def index(request):
     return render(request, 'travel_app/main.html')
 
 
 def coord(request):
     """
-    View added for checking of interaction of map from landing page with 
-    server.
-    :param request: 
-    :return: 
+    This view receives coordinates of location from the GET request, sends it 
+    to the Open Maps API. Than returns name of the location received from API 
+    to the frontend.
+    
+    :param request: data received from frontend, containing longitude and
+      latitude
+    :return: name of location gotten from open maps API
     """
 
-    """TODO RomanPryima: make function sending coordinates and returning 
-    to the frontend name of location received from map API"""
+    # TO DO RomanPryima: move url and api key into cfg file and make view
+    # getting them with cfg parser
 
-    response = request.GET.get('lon') + " " + request.GET.get('lat')
+    api_url = 'http://open.mapquestapi.com/geocoding/v1/reverse'
+    api_key = 'lrGqjWqt82ZGMvQPA0SPXqctacNvptot'
```

Comment:
Btw, I know one more convenient url for reverse geocoding, no api key needed: `http://nominatim.openstreetmap.org/reverse?format=json&lat=49.863280&lon=24.031760&zoom=18&addressdetails=1`